docs/local_usage.md

```diff
@@ -0,0 +1,48 @@
+# Running the bridge locally and testing it end to end manually from UI yourself
```

Comment:
You'll need to add header to this file for it to appear in the generated docs. See `docs/about.md` for an example.